scripts/Text Grammar Corrector/grammer bot.py

```diff
@@ -0,0 +1,47 @@
+# Imports
+import language_tool_python
+
+# Initial API
+tool = language_tool_python.LanguageToolPublicAPI("en-US")
+
+# initalizae filename
+FILENAME = "input.txt"
+
+text = """ update your text here. tool will will fidn grammatically isues and fix it. you can use this text too see an few of of the problems that LanguageTool can detecd. What do you thinks of grammar checkers? Please not that they are not perfect."""
```

Comment:
`fidn` should be find

Reply:
The text is written with bugs on purpose. The script would correct it